zerver/views/streams.py

```diff
@@ -74,6 +76,42 @@ def add_default_stream(request, user_profile, stream_name=REQ()):
 
 @require_realm_admin
 @has_request_variables
+def create_default_stream_group(request, user_profile, group_name=REQ(),
+                                stream_names=REQ(validator=check_list(check_string))):
+    # type: (HttpRequest, UserProfile, Text, List[Text]) -> HttpResponse
```

Comment:
Since you're adding new code, you might want to use the Python 3 typing syntax.

`zerver/management/` has been converted, if you're looking for an example.

(Doesn't super matter for this, but we do want to start switching habits).